Resources/License.rtf

```diff
@@ -1,427 +1,1148 @@
-{\pard \ql \f0 \sa180 \li0 \fi0 END OF TERMS AND CONDITIONS\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 APPENDIX: How to apply the Apache License to your work.\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 \f1   To apply the Apache License to your work, attach the following\line
-  boilerplate notice, with the fields enclosed by brackets "[]"\line
-  replaced with your own identifying information. (Don't include\line
-  the brackets!)  The text should be enclosed in the appropriate\line
-  comment syntax for the file format. We also recommend that a\line
-  file or class name and description of purpose be included on the\line
-  same "printed page" as the copyright notice for easier\line
-  identification within third-party archives.\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 Copyright [yyyy] [name of copyright owner]\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 Licensed under the Apache License, Version 2.0 (the \u8220"License\u8221"); you may not use this file except in compliance with the License. You may obtain a copy of the License at\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 \f1    http://www.apache.org/licenses/LICENSE-2.0\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \u8220"AS IS\u8221" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 \b \fs32 nanopb\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 Copyright (c) 2011 Petteri Aimonen <jpa at nanopb.mail.kapsi.fi>\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 This software is provided \u8216'as-is\u8217', without any express or implied warranty. In no event will the authors be held liable for any damages arising from the use of this software.\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 Permission is granted to anyone to use this software for any purpose, including commercial applications, and to alter it and redistribute it freely, subject to the following restrictions:\par}
-{\pard \ql \f0 \sa180 \li360 \fi-360 1.\tx360\tab The origin of this software must not be misrepresented; you must not claim that you wrote the original software. If you use this software in a product, an acknowledgment in the product documentation would be appreciated but is not required.\par}
-{\pard \ql \f0 \sa180 \li360 \fi-360 2.\tx360\tab Altered source versions must be plainly marked as such, and must not be misrepresented as being the original software.\par}
-{\pard \ql \f0 \sa180 \li360 \fi-360 3.\tx360\tab This notice may not be removed or altered from any source distribution.\sa180\par}
-{\pard \ql \f0 \sa180 \li0 \fi0 Generated by CocoaPods - https://cocoapods.org\par}
+{\rtf1\ansi\ansicpg1252\cocoartf2576
```

Comment:
Hmmm why did this text file change so drastically?

Reply:
We added Firebase to the License file